chebi-update/src/main/java/org/reactome/release/chebiupdate/ChebiUpdater.java

```diff
@@ -450,218 +464,61 @@ public void checkForDuplicates() throws SQLException, Exception
 				+ "where ReferenceDatabase_2_name.name = 'ChEBI' and identifier is not null\n" + "group by ReferenceEntity.identifier\n"
 				+ "having count(ReferenceMolecule.DB_ID) > 1;\n";
```

Comment:
Why is this bare query used?

Reply:
I think this type of query is too complex for the Reactome Java API.